docs/changelog.md

```diff
@@ -19,6 +19,9 @@ permalink: /changelog/
 * **feature-awesomebar**
  * Search engine suggestions will only be displayed if the user inputs at least 2 characters and matches the starting characters of the search engine name. [bug #1851012](https://bugzilla.mozilla.org/show_bug.cgi?id=1851012) 
 
+* **support-ktx**
+  * ⚠️ **This is a breaking change**: the `enterToImmersiveMode()` in `Activity.kt` has been renamed to `enterImmersiveMode()`.
```

Comment:
Why? 🙂

Reply:
I find `enterImmersiveMode()` to be a more concise version of `enterToImmersiveMode()`, and it also aligns more with the `exitImmersiveMode()` name, don't you think so?

Reply:
Renames like this and the `createWindowInsetsController` may be useful to do in a separate commits to make reviewing the non-renamed code easier. 

I do prefer the rename but whether it's worth the change is subjective 🙂